source/agora/common/Config.d

```diff
@@ -102,6 +102,13 @@ public struct NodeConfig
     /// in place of the built-in genesis block as defined by CoinNet
     public string genesis_block;
 
+    /// The Unix timestamp of the Genesis block. Blocks are created at a
+    /// regular interval starting from this time.
+    public time_t genesis_start_time = 1596179709;
```

Comment:
Yep. Also I'm an not sure whether we need or don't need a timestamp in the block header yet. It might be possible to omit it for now, since the estimated block timestamp is derived from its height.